package.json

```diff
@@ -3,30 +3,15 @@
   "version": "2.4.0",
   "description": "Ambianic UI is the main user interface to Ambianic Edge deployments.",
   "author": "Ivelin Ivanov",
-  "license": "Apache-2.0",
-  "homepage": "https://ambianic.ai",
-  "keywords": [
-    "Home Automation",
-    "Ambient Intelligence",
-    "AI",
-    "Business Automation",
-    "Workspace Automation"
-  ],
-  "private": false,
-  "engines": {
-    "node": ">=12.10.0"
-  },
   "scripts": {
     "serve": "vue-cli-service serve",
     "build": "vue-cli-service build",
+    "test:unit": "vue-cli-service test:unit",
     "lint": "vue-cli-service lint",
-    "test": "npx cypress run",
```

Comment:
@BKristenssonAlfsson Looks like you replaced cypress with jest in the test target. Should we instead run both unit and e2e tests sequentially in the test target and have separate targets `test:jest` and `test:e2e` for individual test type runs?